services/core-api/app/api/now_applications/models/administrative_amendments/application_reason_code_xref.py

```diff
@@ -0,0 +1,23 @@
+from sqlalchemy.schema import FetchedValue
+from sqlalchemy.dialects.postgresql import UUID
+
+from app.extensions import db
+from app.api.utils.models_mixins import Base
+from app.api.constants import NOW_APPLICATION_EDIT_GROUP
+
+
+class ApplicationReasonXref(Base):
+    __tablename__ = "application_reason_code_xref"
+
+    _edit_groups = [NOW_APPLICATION_EDIT_GROUP]
+    _edit_key = NOW_APPLICATION_EDIT_GROUP
+
+    now_application_id = db.Column(
+        db.Integer, db.ForeignKey('now_application.now_application_id'), primary_key=True)
+    application_reason_code = db.Column(
+        db.String(3),
+        db.ForeignKey('application_reason_code.application_reason_code'),
+        primary_key=True)
+
+    def __repr__(self):
```

Comment:
For all the new models made, can we use `self.__class__.__name__` to get the class name instead in the `repr` functions

Reply:
done, I think we may need to remind about this on one of our technical meetings, I didn't know that we have this agreement